packages/lesswrong/lib/crud/withSingle.ts

```diff
@@ -119,7 +119,7 @@ export function useSingle<FragmentTypeName extends keyof FragmentTypes>({
   extraVariablesValues?: any,
   skip?: boolean,
 }): {
-  document: FragmentTypes[FragmentTypeName]
+  document: FragmentTypes[FragmentTypeName],
```

Comment:
man how was this even working before?

Reply:
Commas are required when entries in an object are on the same line, but optional (in the same way semicolons are optional) when split between lines.)